content/en/docs/concepts/services-networking/service.md

```diff
@@ -1184,6 +1179,34 @@ You can use TCP for any kind of Service, and it's the default network protocol.
 You can use UDP for most Services. For type=LoadBalancer Services, UDP support
 depends on the cloud provider offering this facility.
 
+### SCTP
+
```

Comment:
Hi @danwinship .
Is it worthwhile to notify readers that this feature is changing to GA in v1.20?
Add a feature state label:
```
{{< feature-state for_k8s_version="v1.20" state="stable" >}}
```

Reply:
I guess so? I've never done this before so I'm not sure what changes we normally do for a beta to GA transition

Reply:
When a feature first moves to GA the label is updated but remains with the content.
Keeping the label in place or removing the label is up to you. Eventually, the label will be removed after
several more releases have completed.